docs/src/man/getting_started.md

```diff
@@ -423,6 +434,20 @@ julia> df[(df.A .> 500) .& (300 .< df.C .< 400), :]
 │ 98  │ 795   │ 8     │ 398   │
 │ 99  │ 797   │ 8     │ 399   │
 ```
+Where a specific subset of values needs to be matched, the `in()` function can be applied:
+
+```jldoctest dataframe
+julia> df[in.(df.A, Ref([1, 5, 601])), :]
+3×3 DataFrame
+│ Row │ A     │ B     │ C     │
+│     │ Int64 │ Int64 │ Int64 │
+├─────┼───────┼───────┼───────┤
+│ 1   │ 1     │ 1     │ 1     │
+│ 2   │ 5     │ 1     │ 3     │
+│ 3   │ 601   │ 7     │ 301   │
+```
+
+Equivalently, the `in` function can be used as a [`Fix2`](https://github.com/JuliaLang/julia/blob/v1.0/base/operators.jl#L964) function object - the above call would then be written as `df[in([1, 5, 601]).(df.A), :]`.
```

Comment:
Seriously, mentioning an unexported object and pointing to the source code in a Getting Started page? :-D

```suggestion
Equivalently, the `in` function can be called with a single argument to create a function object that tests whether each value belongs to the subset (partial application of `in`): `df[in([1, 5, 601]).(df.A), :]`.
```

Reply:
Well the link I have given is exported :), but your proposal is better.